### PR TITLE
Recommend modifying .bashrc in pipvirtualenv/setup.html

### LINKED ISSUE #1629: Recommend modifying .bashrc instead of running source .bash_profile in the pipvirtualenv mission
Hi!

In the [Running into problems](https://github.com/openhatch/oh-mainline/blob/master/mysite/missions/templates/missions/pipvirtualenv/setup.html#L49) section of [the pipvirtualenv mission](http://openhatch.org/missions/pipvirtualenv/setup), we recommend running ```source ~/.bash_profile``` if pip does not work right away after running the [pip2014](http://pip2014.com/) script and restarting the terminal. I think this has to do something with .bash_profile not getting sourced even after the terminal is restarted. After taking a look at how the pip2014 script modifies .bash_profile, I found that it would be better if we recommend running the following commands instead:

```
echo source "${HOME}/.pip_bootstrap_profile.sh" >> ~/.bashrc
source ~/.bashrc
```

I tested this and after running this command, pip and the other tools installed by pip2014 should work; because according to this [page](http://superuser.com/questions/49289/what-is-the-bashrc-file), whenever an interactive shell that is not a login shell is started, bash reads and executes commands from ~/.bashrc, if that file exists. :-)

Thanks & regards,
Eeshan Garg

### PR DESCRIPTION
**Pull request by eeshangarg**
Closes #1629 :-)

### CODE REVIEW

File: mysite/missions/templates/missions/pipvirtualenv/setup.html [hunk 1]
```diff
@@ -58,8 +58,9 @@
     command-line) after restarting the terminal, the shell replies with
     <q>command not found</q> or something similar.</p>
 
-    <p>If this is the case, the following command should fix it:</p>
-    <pre>source ~/.bash_profile</pre>
+    <p>If this is the case, the following commands should fix it:</p>
+    <pre>echo source "${HOME}/.pip_bootstrap_profile.sh" >> ~/.bashrc</pre>
```
Comment (willingc) [inline mysite/missions/templates/missions/pipvirtualenv/setup.html]: @eeshangarg Howdy :) Would you mind double checking that Line 62 does not overwrite if a user already has an existing .bashrc?

Comment (paulproteus) [inline mysite/missions/templates/missions/pipvirtualenv/setup.html]: FWIW, reading it, I can see that it uses >> not >, and therefore avoids
overwriting.​

Comment (eeshangarg) [inline mysite/missions/templates/missions/pipvirtualenv/setup.html]: Hi @paulproteus, @willingc! I have already tested this with an existing .bashrc file and this command does not overwrite the existing .bashrc. :-)

Comment (willingc) [inline mysite/missions/templates/missions/pipvirtualenv/setup.html]: Thank you both :) I will merge.

**Comment by willingc** [comment]
Thank you @eeshangarg :penguin: :guitar:

**Pull request merged**